proxy/coco/src/control.rs

```diff
@@ -10,12 +10,12 @@ use radicle_surf::vcs::git::git2;
 
 use crate::{config, error::Error, project, signer, state::State, user::User};
 
-/// Deletes the local git repsoitory coco uses to keep its state.
+/// Deletes the local git repsoitory coco uses for its state.
 ///
 /// # Errors
 ///
-/// Will error in case the call to the [`std::fs::remove_dir_all`] fails.
-pub fn nuke_monorepo() -> Result<(), std::io::Error> {
+/// * if the call to [`std::fs::remove_dir_all`] fails.
+pub fn wipe_monorepo() -> Result<(), std::io::Error> {
```

Comment:
Why is _this_ one called wipe and not reset? :sweat_smile:

Reply:
Yeah, was also wondering. @xla?

I could still rename it in my branch...

Reply:
:shrug: - creative freedom.

Reply:
:art:

Reply:
![freedom](https://thumbs.gfycat.com/ElegantEsteemedLacewing-size_restricted.gif)